pages/common/tar.md

```diff
@@ -30,3 +30,7 @@
 - List the contents of a tar file:
 
 `tar tvf {{source.tar}}`
+
+- Extract files matching a pattern:
+
+`tar xf {{source.tar}} --wildcards "*.html"`
```

Comment:
Erm oops! I meant to add a small comment mentioning the `*.html`. Perhaps it could be wrapped in `{{` and `}}`?

Reply:
Sure!

Reply:
Excellent, thanks, @amsross! Once we get a second approval, we'll get to merging :D